<launch>
  <remap from="scan" to="/p3dx/laser" />
  <node pkg="lms1xx" type="LMS1xx_node" name="LMS1xx_node">
    <remap from="scan" to="/p3dx/laser" />
    <param name="host" value="169.254.63.213" />
  </node>
  <!-- Laser tf frame base_link->laser -->
  <node pkg="pioneer_control" type="laser_tf_broadcaster.py" 
    name="laser_tf" />
</launch>
